op-chain-ops/deployer/state/intent.go

```diff
@@ -15,7 +15,7 @@ var emptyAddress common.Address
 type Intent struct {
 	L1ChainID uint64 `json:"l1ChainID" toml:"l1ChainID"`
 
-	SuperchainRoles SuperchainRoles `json:"superchainRoles" toml:"superchainRoles"`
+	SuperchainRoles SuperchainRoles `json:"-" toml:"-"`
```

Comment:
I'm not sure if/when we ever serialize as JSON, but figured we should keep the JSON and TOML behavior consistent

Reply:
Interesting, I had to read what the [hyphen means ](https://pkg.go.dev/encoding/json#Marshal) in this scenario but makes sense to me.